readme.html

```diff
@@ -53,5 +53,118 @@ <h2>Pagination</h2>
         <li>This will fetch all 950 items</li>
     </ul>
 </ul>
+<h2>Port Information</h2>
+    <p>
+        The app uses HTTP/ HTTPS protocol for communicating with the Okta server. Below are the default ports used by Splunk SOAR.
+        <table>
+            <tr class=plain>
+                <th>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Service Name</th>
+                <th>Transport Protocol</th>
+                <th>Port</th>
+            </tr>
+            <tr>
+                <td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;http</td>
+                <td>tcp</td>
+                <td>80</td>
+            </tr>
+            <tr>
+                <td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;https</td>
+                <td>tcp</td>
+                <td>443</td>
+            </tr>
+        </table>
+    </p>
+    <p>
+        <h2>Permissions</h2>
+        <p>
+            Super admins can assign admin permissions to individuals or groups.
+            <br>
+            <b>Individual</b> assignments are better for a manageable number of admin accounts. When you assign admin permissions to individuals, you do so one at a time, whenever necessary. Remember that every org must always have at least one individually assigned super admin.
```

Comment:
at least one individual assigned super admin

Reply:
In the okta app, at least one user must be assigned as a Super Administrator (SUPER_ADMIN) role.